geoportal/geoportailv3_geoportal/static-ngeo/js/catalog/CatalogController.js

```diff
@@ -135,15 +142,23 @@ exports.prototype.setThemeZooms = function(tree) {
       var resolutions = tree['metadata']['resolutions'];
       maxZoom = resolutions.length + 7;
     }
+  
     var map = this['map'];
     var currentView = map.getView();
+
+    let rotation = 0;
+    if (this.ngeoLocation_.getParam('rotation') !== undefined) {
+      rotation = this.ngeoLocation_.getParam('rotation');
+    }
+
     map.setView(new olView({
       maxZoom: maxZoom,
       minZoom: 8,
       extent: this.maxExtent_,
       center: currentView.getCenter(),
       enableRotation: true,
-      zoom: currentView.getZoom()
+      zoom: currentView.getZoom(),
+      rotation,
```

Comment:
why not like `rotation: rotation` ?
(but I must admit, I don't really know the application context, my comment is based on similarity...)

Reply:
it's es6 syntaxic sugar: when the key has the same name as the variable, you can shortcut like this.

Reply:
When using ES6 syntax, you can skip the redendancy of writing key: value when it is the same to just write the key. The compiler will interpret the parameter key as the same as the value.

Reply:
prem's (-;

Reply:
:-(

Reply:
ne vous battez pas tout de meme.
thanks for the clarifications